### ISSUE TITLE
Deprecation of Meetup API v2 - migrate till August 14!

### ISSUE THREAD

**Issue by TomasVotruba** [TomasVotruba/friendsofphp.org#133]
**tl;dr; Switch from API Key to OAuth untill August 14, or it will be very expensive to get any data.**

- API Keys will be replaced by OAuth: We will be removing API keys on August 15, 2019 and requiring you to [authenticate with OAuth](https://www.meetup.com/meetup_api/auth/?utm_medium=email&utm_source=transactional&utm_campaign=api_update&utm_content=en_06142019).

- Move to OAuth soon for continued free API access: Until August 15, members will be able [to apply for OAuth access](https://www.meetup.com/meetup_api/auth/?utm_medium=email&utm_source=transactional&utm_campaign=api_update&utm_content=en_06142019) free of charge. After August 15, anyone who wants to apply for API access through OAuth will need to have a [Meetup Pro account](https://www.meetup.com/pro/?utm_medium=email&utm_source=transactional&utm_campaign=api_update&utm_content=en_06142019) in order to do so.

- We’re removing API version 2: Before August 15, you will need to switch to [API version 3](https://www.meetup.com/meetup_api/?utm_medium=email&utm_source=transactional&utm_campaign=api_update&utm_content=en_06142019). API version 2 will no longer be available. You can find [more information on our API updates](https://help.meetup.com/hc/en-us/articles/360028901812?utm_medium=email&utm_source=transactional&utm_campaign=api_update&utm_content=en_06142019) and answers to common questions on our Help Center.
